cmd/make_dbscale_sandbox.go

```diff
@@ -57,4 +64,21 @@ func main() {
 
 	utils.MySQLInstallReplication(*mysqlDirPath, rep1Dir, start_port)
 	utils.MySQLInstallReplication(*mysqlDirPath, rep2Dir, start_port+2)
+
```

Comment:
in main function just call utils.MySQLInstallGrantFile,
all other init operations you can be done in mysql.go.
So that you can make the main function clearly.